.checkov.baseline

```diff
@@ -168,6 +168,12 @@
                     "check_ids": [
                         "CKV_AWS_158"
                     ]
+                },
```

Comment:
Adding this since the checkov were failing for the ECS log groups ( outputing that the ECS logs grooup need to be encrypted by KMS ). All ECS log groups are not encrypted rightnow and can be taken as another security task . Out of scope for this enhancement feature